test/test-get-passwd.c

```diff
@@ -24,6 +24,11 @@
 #include <string.h>
 
 TEST_IMPL(get_passwd) {
+//  TODO fix test on qemu
```

Comment:
```suggestion
/* TODO(gengjiawen): Fix test on QEMU. */
```

Can you update the other comments as well.